test/unit/account/search_test.rb

```diff
@@ -161,6 +161,7 @@ class IncrementalIndexingTest < ActiveSupport::TestCase
       ThinkingSphinx::Test.clear
       ThinkingSphinx::Test.init
       ThinkingSphinx::Test.start index: false
+      ThinkingSphinx::Test.enable_search_jobs!
```

Comment:
Don't you need to disable on `teardown`?

Reply:
In `setup` these are disabled. So it is redundant to disable also in `teardown`. Moreover mocks are automatically removed on teardown anyway. So if we disable in teardown, then they will be automatically enabled by the mocha `teardown`.

Reply:
which `setup` do you mean?

Reply:
https://github.com/3scale/porta/pull/3492/files#diff-33d4a34306c39c3dc095690ac70832e3a35f67a38cc820ceeae333006b330513R55